.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+node_js:
+  - '12'
```

Comment:
Lets migrate to https://github.com/antongolub/blank-ts-repo/blob/master/.travis.yml